rdmd.d

```diff
@@ -831,23 +831,26 @@ string innerEvalCode(string[] eval)
     import std.string : join, stripRight;
     // assumeSafeAppend just to avoid unnecessary reallocation
     string code = eval.join("\n").stripRight.assumeSafeAppend;
-    if (code.length > 0 && code[$ - 1] != ';')
-        code ~= ';';
+    // Revert https://github.com/dlang/tools/pull/303 until
+    // https://github.com/dlang/tools/pull/317 has been resolved
+    //if (code.length > 0 && code[$ - 1] != ';')
+        //code ~= ';';
```

Comment:
This isn't a reversion of #303, it's a (breaking) change relative to _previous_ behaviour.

Until #303, a terminating semicolon would _always_ be added.  What #303 changed was to stop that happening in cases where there was _already_ a terminating semicolon.